manifests/agent.pp

```diff
@@ -34,6 +34,16 @@
       require => Package['logentries-daemon'],
       notify  => Service['logentries'],
     }
+  } elsif ( $logentries::params::agent_key == '' ) and ( $logentries::params::register == false ) {
```

Comment:
Will this interfere with cases where Datahub is used?

In those cases, the agent_key would be empty and registration would also be false.